scene/main/viewport.cpp

```diff
@@ -2948,12 +2948,14 @@ Control *Viewport::get_modal_stack_top() const {
 }
 
 String Viewport::get_configuration_warning() const {
-
 	/*if (get_parent() && !Object::cast_to<Control>(get_parent()) && !render_target) {
 
 		return TTR("This viewport is not set as render target. If you intend for it to display its contents directly to the screen, make it a child of a Control so it can obtain a size. Otherwise, make it a RenderTarget and assign its internal texture to some node for display.");
 	}*/
 
+	if (size.x == 0 || size.y == 0) {
+		return TTR("This viewport can't render anything.\nConsider increasing the size.");
```

Comment:
This should be explicit that the reason it can't render anything is because of the size and size needs to be greater than 0.

Consider: "Viewport size must be greater than 0 to render anything."